.github/workflows/ci.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
             executable_path: 'dist\kubemarine.exe'
           - target: {os: ubuntu-latest, arch: 'linux-x86_64'}
             build_cmd: >
-              docker run -t -v $(pwd):/opt/app-root/src centos/python-38-centos7 bash -c
+              docker run -t -v $(pwd):/app bitnami/python:3.11.2-debian-11-r15 bash -c
```

Comment:
Executable file created this way will not work on Centos 7 and probably some other relatively old OS

Reply:
Yes, the problem is relevant
I used an old docker image